mono-traversable/ChangeLog.md

```diff
@@ -1,3 +1,7 @@
+## next
```

Comment:
Do you mind setting this to 1.0.6.0 and then adding a `@since` comment and a package.yaml version bump.

Reply:
Sure, I'll go ahead and do that.

Reply:
Fixed in commit 5dd21c1.